test/setup_sbt.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+
+load "setup_sbt.sh"
+load "test_helper/bats-support/load"
+load "test_helper/bats-assert/load"
+
+TMPDIR=""
+
+function setup {
+  TMPDIR=$(mktemp -d)
+  mkdir -p $TMPDIR/.sbt/1.0/plugins
```

Comment:
This might be why your script isn't failing (my above comment).  Why is this here?

Reply:
See [my](https://github.com/blackboard-innersource/gh-action-setup-artifactory/pull/21#discussion_r1785665252). I guess we don't care about Scala SBT installation in this script and this folder should be created when you install SBT.  So for testing purposes, I'll have to create it manually.

Reply:
I removed the creation of `.sbt/1.0/plugins` directory from test and added a check/creation into the main script.